django/conf/locale/__init__.py

```diff
@@ -536,6 +536,12 @@
         'name': 'Urdu',
         'name_local': 'اردو',
     },
+    'uz': {
+        'bidi': False,
+        'code': 'uz',
+        'name': 'Uzbek',
+        'name_local': 'oʻzbek tili',
```

Comment:
I think we should drop `tili`, we don't have a `language` suffix for other languages.

Reply:
@claudep, @felixxm, No, I believe it is better to leave as it is with the suffix `tili` for the name. The reason why it is actually added is that the meaning of the word `o'zbek` is "uzbek person" and only when the suffix is added it is meant like a language. The same works for other languages when they are translated into Uzbek so that all the names without `tili` means the person or the native speaker of that language. As fas as I know, it is from my language learning experience :)

Reply:
@bedilbek Thanks for clarification :+1:

Reply:
@felixxm you are welcome :smile: